en/snap/platform/overview.md

```diff
@@ -135,30 +139,30 @@ For example:
 	- [Card Payment](https://www.shopify.com/login?redirect=%2Fadmin%2Fauthorize_gateway%2F1055299)
 	- [Card Installment Payment](https://www.shopify.com/login?redirect=%2Fadmin%2Fauthorize_gateway%2F1055413)
 
-2. Login with your Shopify account when prompted, then you will need to click **Install** on the payment gateway:
+2. When prompted, login with your Shopify account. Click **Install** on the payment gateway.
 ![shopify](./../../../asset/image/shopify-13.png ':size=400')
 
-3. You will be navigated to payment settings page (`Settings > Payments` menu on your Shopify admin page). Click **Choose Alternative Payments**
+3. On Shopify adminstration page, go to **Settings > Payments**. Click **Choose Alternative Payments**.
```

Comment:
The navigation did **happen automatically**, so we should not change it to instruct them navigate manually.